ui/components/ui/loading-screen/loading-screen.component.js

```diff
@@ -1,49 +1,39 @@
-import React, { Component, isValidElement } from 'react';
+import React, { isValidElement } from 'react';
 import PropTypes from 'prop-types';
 import Spinner from '../spinner';
 
-class LoadingScreen extends Component {
-  static defaultProps = {
-    loadingMessage: null,
-    showLoadingSpinner: true,
-  };
-
-  static propTypes = {
-    loadingMessage: PropTypes.oneOfType([PropTypes.string, PropTypes.element]),
-    showLoadingSpinner: PropTypes.bool,
-    header: PropTypes.element,
-  };
-
-  renderMessage() {
-    const { loadingMessage } = this.props;
-
-    if (!loadingMessage) {
-      return null;
-    }
-
-    return isValidElement(loadingMessage) ? (
-      loadingMessage
-    ) : (
-      <span>{loadingMessage}</span>
-    );
+const renderMessage = (message) => {
```

Comment:
I think this could be brought into the top level of the component instead of extracted.
```
let messageElement = null;
if (message) {
  messageElement = isValidElement(message) ? message : <span>{message}</span>;
}
```

Reply:
Thanks for the callout! agreed. updated https://github.com/MetaMask/metamask-extension/pull/17935/commits/3e03c350eb7b0bf088367dbbb5e0601772a127f0